src/transformers/models/wav2vec2/modeling_flax_wav2vec2.py

```diff
@@ -1235,7 +1235,7 @@ def __call__(
         deterministic: bool = True,
         output_attentions=None,
         output_hidden_states=None,
-        freeze_feature_enocder=False,
+        freeze_feature_encoder=False,
```

Comment:
Good catch! Thankfully this didn't cause any issue as we don't use the keyword argument in `apply`.